test/integration_test.rb

```diff
@@ -23,6 +23,12 @@ def test_no_test_is_selected
         assert_equal "", output[:stdout]
       end
 
+      def test_all_tests_are_selected_without_mapping
+        git_rm_and_commit("#{@repo.workdir}/.ttnt", 'Remove .ttnt')
+        output = rake('ttnt:test:run')
+        assert_match '2 runs, 6 assertions, 0 failures', output[:stdout]
```

Comment:
To avoid having to change this anytime you want to add a new test to this dummy test suite, you can do something along the lines of:

~~~ruby
def test_all_tests_are_selected_without_mapping
  git_rm_and_commit("#{@repo.workdir}/.ttnt", 'Remove .ttnt')

  rake_ttnt_result = rake('ttnt:test:run')[:stdout].lines.last
  rake_test_result = rake('test')[:stdout].lines.last

  assert_equal rake_test_result, rake_ttnt_result
end
~~~